bot/converters.py

```diff
@@ -574,6 +574,55 @@ async def convert(self, ctx: Context, arg: str) -> t.Optional[dict]:
                 raise e
 
 
+class DayDuration(Converter):
```

Comment:
We should probably add a typehint for this converter below the class inside the `if t.TYPE_CHECKING:` (putting comment here as GitHub needs it to be on a changed line).